OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -136,51 +412,171 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        It might have been written in the same hand as that of <ref target="#a2"/>.
+                                    </note>
+                                </item>
+                                <item xml:id="a4">
+                                    <desc type="GuestText" xml:id="LIT5921SalamLaki"/>
+                                    <locus from="80va" to="80vb"/>
+                                    <q xml:lang="gez">
+                                        <seg part="I">
+                                            ሰላም፡ ለኪ፡ ደብረ፡ ሊባኖስ፡ ግነት፡ ፄና፡ መዓዛኪ፡ ዕፍረት፡ ዘአዕረፈ፡ ብኪ፡ ተክለ፡ <choice><sic>ሃማኖተ፡</sic><corr>ሃይማኖት፡</corr></choice>
+                                        </seg>
+                                    </q>
+                                   <note>Part of the text is hardly readable due to faded ink. Spaces for rubrication have been left unfilled.</note>
+                                </item>
+                                
+                                <item xml:id="e1">
+                                    <desc>A white sticker is glued to the inner side of the front board. The name of <persName ref="PRS5782JuelJen"/> is printed on it
+                                        (<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>) and the signatures "MS. Aeth. e. 38" and "MS. Aethiop. J-J. 41"are written in
+                                        pencil. Below is glued an additional sticker, with the signature "Ms 41"printed on it.</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc>Writings in pencil are drawn up on the recto side of the protective bifolium glued to the front cover.
+                                        <q xml:lang="en">81 folios. 10 full page paintings. 2 sketches. size 7" x 5"</q>.</desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <note>Aids for identifying weekday readings are written in the upper margin within a frame decorated in black and red: 
+                                        <q xml:lang="gez">ዘሠሉስ።</q> (<locus target="#26ra"/>), <q xml:lang="gez">ዘረቡዕ።</q> (<locus target="#35ra"/>), 
+                                        <q xml:lang="gez">ዘሐሙስ።</q> (<locus target="#45rb"/>), <q xml:lang="gez">ዘዓርብ።</q> (<locus target="#54ra"/>), 
+                                        <q xml:lang="gez">ዘቀዳሚት።</q> (<locus target="#60va"/>), <q xml:lang="gez">ዘእሑድ።</q> (<locus target="#66vb"/>), 
+                                    </note>
+                                </item>
+                                <item xml:id="e4">
+                                    <note>The text is written on erasure on <locus target="#12ra #39rb #44rb"/>. 
```

Comment:
This could be formulated better to avoid misunderstandings. "Corrections over erasures..."